### PR TITLE
Presentation session-19-fluent.md

### PR DESCRIPTION
**Pull request by SepidehRahimiii**

### CODE REVIEW

File: 5th-nextgen/session-19/session-19-fluent.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+print('Numbers')
+print(' str :', re_numbers_str.findall(text_str))
+```
+
+## str Versus bytes in os Functions
+The GNU/Linux kernel is not Unicode savvy, so in the real world you may find file‐
+names made of byte sequences that are not valid in any sensible encoding scheme,
+and cannot be decoded to str.
+
+**Example:**
```
Comment (ghzamani) [inline 5th-nextgen/session-19/session-19-fluent.md:117]: It seems you forgot to put ```python

Comment (SepidehRahimiii) [reply 5th-nextgen/session-19/session-19-fluent.md:117]: yes applied the changes .

**Review by Amirjahandar — APPROVED** [review]
confrimed

**Review by Amirjahandar — APPROVED** [review]

File: 5th-nextgen/session-19/session-19-fluent.md (hunk 1 above)
Comment (Amirjahandar) [reply 5th-nextgen/session-19/session-19-fluent.md:117]: so i check it

**Review by Amirjahandar — APPROVED** [review]

**Comment by Amirjahandar** [comment]
@ghzamani 
miss zamani ,
the problem was resolved please review code

**Review by Amirjahandar — APPROVED** [review]

**Pull request merged**